src/components/TagItem.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+	<div class="tag-group">
+		<button class="tag-group__label"
+			:style="{
+				color: convertHex(tag.color, 1),
+				'background-color': convertHex(tag.color, 0.15)
+			}">
+			{{ tag.displayName }}
+		</button>
+		<Actions :force-menu="true">
+			<NcActionButton v-if="renameTagLabel"
+				@click="openEditTag">
+				{{ t('mail','Edit name or color') }}
+			</NcActionButton>
+			<ActionInput v-if="renameTagInput"
+				:value="tag.displayName"
+				@submit="renameTag(tag, $event)" />
+			<ActionText
+				v-if="showSaving">
+				<template #icon>
+					<IconLoading :size="20" />
+				</template>
+				{{ t('mail', 'Saving new tag name …') }}
+			</ActionText>
+
+			<NcColorPicker
+				class="app-navigation-entry-bullet-wrapper"
+				:value="`#${tag.color}`"
+				@input="updateColor">
+				<div :style="{ backgroundColor: getColor }" class="color0 app-navigation-entry-bullet" />
```

Comment:
```suggestion
				<div :style="{ backgroundColor: tag.color }" class="color0 app-navigation-entry-bullet" />
```

Should work as well. Then you can remove getColor method and editColor.